### DISCUSSION TITLE
Is there any real difference between using ArcWelder on Octoprint or on Cura?

### DISCUSSION THREAD

**Discussion by apbarratt** [FormerLurker/ArcWelderPlugin#198] [Q&A]
I have been using ArcWelder in Cura for a couple of days before realising it was also an Octoprint plugin.  I'm wondering if there's really much difference in either option?

What would be the pros and cons of using either of them?

**Comment by seantapscott** [comment]
I think the plugin has more features, more options, and gets updated more
frequently.  The Cura version is more difficult to manage.

**Reply by apbarratt** [reply]
Thank you :) I think I'll ditch the Cura version then :)

**Comment by XxxMikeyxxX** [comment]
I like running it on the PI as I use different slicers and don't want to try to manage all the snapin/plugins that would add up.
